packages/ddp-client/livedata_connection.js

```diff
@@ -38,7 +38,11 @@ var Connection = function (url, options) {
     reloadWithOutstanding: false,
     supportedDDPVersions: DDPCommon.SUPPORTED_DDP_VERSIONS,
     retry: true,
-    respondToPings: true
+    respondToPings: true,
+    // When updates are coming within this ms interval, batch them together.
+    bufferedWritesInterval: 5,
+    // Flush buffers immediately if writes are happening continuously for more than this many ms.
+    bufferedWritesMaxAge: 500
```

Comment:
Yeah I think having a setting for it would be good. We can either put it in Meteor.settings, or have a runtime function call for it (which makes sense to me since it would be totally reasonable to want to change the delay on certain pages of your app that want to be more realtime).

Perhaps something like `DDPClient.setBufferMaxAge`?